torch/_dynamo/variables/builtin.py

```diff
@@ -1294,6 +1312,11 @@ def call_id(self, tx, *args):
             mod = tx.output.get_submodule(nn_mod_variable.module_key)
             return variables.ConstantVariable(id(mod))
         else:
+            try:
+                value = args[0].as_python_constant()
+                return variables.ConstantVariable(id(value))
+            except:
+                pass
             unimplemented(f"call_id with args {args}")
```

Comment:
Couldn't this unimplemented method be raised from the above exception so that we know why the args did not convert?